### ISSUE TITLE
Raw LIDAR scans

### ISSUE THREAD

**Issue by aserbremen** [NeBula-Autonomy/nebula-multirobot-dataset#2]
Thank you for your great work and release of source code datasets for multirobot SLAM. I was wondering whether there is a plan to also release the raw lidar point clouds. Since preprocessing LIDAR point clouds and choosing keyframes can differ for each mutlirobot SLAM algorithm and impact performance. For comparability and research the raw point clouds might be very helpful.

**Comment by ChanJoon** [comment]
+1

**Comment by BenjaminMorrell** [comment]
@yunzc - these should all be available here do you think? https://github.com/NeBula-Autonomy/nebula-odometry-dataset
But maybe not from every agent. That could be used as a starting point. But for completeness, we will need to look into that.

**Comment by ychang-bdai** [comment]
@BenjaminMorrell agreed that those should cover the urban, part of the tunnel and the SubT finals sequences. But I think the odometry datasets are also not online right now. Do you guys have backups on your end perhaps?

**Comment by yunzc** [comment]
@BenjaminMorrell agreed that those should cover the urban, part of the tunnel and the SubT finals sequences. But I think the odometry datasets are also not online right now. Do you guys have backups on your end perhaps?

**Comment by ChanJoon** [comment]
> @yunzc - these should all be available here do you think? [NeBula-Autonomy/nebula-odometry-dataset](https://github.com/NeBula-Autonomy/nebula-odometry-dataset) But maybe not from every agent. That could be used as a starting point. But for completeness, we will need to look into that.

I also wonder you have access to those dataset link. Currently it's not working.

**Comment by yunzc** [comment]
> > @yunzc - these should all be available here do you think? [NeBula-Autonomy/nebula-odometry-dataset](https://github.com/NeBula-Autonomy/nebula-odometry-dataset) But maybe not from every agent. That could be used as a starting point. But for completeness, we will need to look into that.
> 
> I also wonder you have access to those dataset link. Currently it's not working.

Unfortunately I don't have it where easily accessible. Hopefully @femust is able to figure out something soon, otherwise I will see what I can do.
